phpcs.xml.dist

```diff
@@ -146,7 +147,7 @@
 	     provided as a comma-delimited list. -->
 	<rule ref="WordPress.WP.I18n">
 		<properties>
-			<property name="text_domain" type="array" value="wporg-learn,wporg_learn" />
+			<property name="text_domain" type="array" value="wporg-learn,wporg_learn,wporg" />
```

Comment:
@iandunn alternatively we could change the references in `functions.php` to `wporg-learn`, I wasn't 100% sure what is correct.

Reply:
Looks like it should be `wporg-learn` for the Learn theme and plugin. `wporg` for mu-plugins. There are a few instances of `wporg` textdomain in the theme's `functions.php`, those are probably mistakes.

It's interesting that the code uses a mishmash of `wporg-learn` and `wporg_learn`. I guess it works but that seems like it must be a mistake? This says textdomains must use dashes not underscores: https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains. I wonder if there's a reason for it.

I'd be tempted to remove `wporg` from the property here and fix those lint errors (there are only a handful). And leave the dash/hyphen thing for a later PR.

Reply:
> Looks like it should be wporg-learn for the Learn theme and plugin. wporg for mu-plugins. There are a few instances of wporg textdomain in the theme's functions.php, those are probably mistakes.

👍 From the search result, `'wporg'` all appears in the files under `mu-plugins/`, I believe the `'wporg'` in `themes/pub/wporg-learn-2020/functions.php` can be changed to `'wporg-learn'` safely.

> It's interesting that the code uses a mishmash of wporg-learn and wporg_learn. I guess it works but that seems like it must be a mistake? This says textdomains must use dashes not underscores: https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains. I wonder if there's a reason for it.

I can find both of them work on https://translate.wordpress.org/locale/zh-tw/default/meta/learn-wordpress/, but not sure why it could work. Since I thought only the text domain specified in the theme [style.css](https://github.com/WordPress/Learn/blob/trunk/wp-content/themes/pub/wporg-learn-2020/style.css#L10) could take effect.

Reply:
I'm assuming something (in core? GlotPress?) magically replaces the underscore with a dash, so it appears to work ok. We should follow the recommendations though.